db/migrate/20210426141920_add_profile_photo_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddProfilePhotoToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :profile_photo, :string, default: "https://cdn.pixabay.com/photo/2015/05/31/10/54/coffee-791045_960_720.jpg"
```

Comment:
I would add this image to your files locally, OR even better add it to you S3 bucket so you don't have to worry about a potentially broken image.